_test/test_utilities/test_warnonce.m

```diff
@@ -6,13 +6,13 @@
         end
 
         function setUp(~)
-            warnonce('clear', 'HORACE:panic:panic')
-            warnonce('clear', 'HORACE:panic:arr')
+            evalc("warnonce('clear', 'HORACE:panic:panic')");
```

Comment:
Question: Why does this need to be changed to being run by `evalc` (and similarly in the rest of `function setUp` and `function tearDown)`? The reason why it is done elsewhere in the tests is to capture the output that would normally be written to the screen by haveing a return argument `out`: e.g. in `out = evalc("warnonce('HORACE:panic:panic', 'Panic at the disco')");`. if the return argument is not needed i don't understand why the `evalc` is needed.